tests/scripts/all.sh

```diff
@@ -4005,14 +4012,45 @@ test_build_opt () {
     done
 }
 
```

Comment:
I would like the new components to print out the compiler version that they're using. Otherwise that information doesn't appear in the CI logs. I think a good way to have the information would be to run `$cc --version` in `test_build_opt`.

Maybe we should also add this information to `output_env.sh`.

Reply:
Will add them.